PowerPointLabs/PowerPointLabs/Ribbon1.cs

```diff
@@ -2056,13 +2134,13 @@ public void TransparentEffectClick(Office.IRibbonControl control)
             TransparentEffect(selection.ShapeRange);
         }
 
-        public void EffectsLabSettingsButtonClick(Office.IRibbonControl control)
+        /*public void EffectsLabSettingsButtonClick(Office.IRibbonControl control)
         {
             var effectsLabSettingsDialog = new EffectsLabSettings(_effectsLabSelectionIsCover);
             effectsLabSettingsDialog.SettingsHandler += EffectsLabSettingChangedHandler;
             effectsLabSettingsDialog.ShowDialog();
             effectsLabSettingsDialog.Dispose();
-        }
+        }*/

```

Comment:
Dead code rarely gets revived. It's best to just delete it. We can always recover from Git if we need to. Having commented out code makes the code looks very unprofessional.